src/mixins/view.js

```diff
@@ -163,11 +167,18 @@ export default {
     // https://github.com/jashkenas/backbone/blob/1.2.3/backbone.js#L1235
     this._removeElement();
 
+    if (shouldTriggerDetach) {
+      this._isAttached = false;
+      this.triggerMethod('detach', this);
+    }
+
     // remove children after the remove to prevent extra paints
     this._removeChildren();
 
     this._destroyBehaviors(args);
 
+    this._isDestroyed = true;
```

Comment:
I don't disagree.

But we should remove the comments such as:
https://github.com/marionettejs/backbone.marionette/blob/next/src/object.js#L44
which indicate the opposite.